libjsqrc/ethereumjs/.travis.yml

```diff
@@ -1,13 +1,17 @@
 language: node_js
 node_js:
+  - "0.12"
   - "0.11"
   - "0.10"
 before_script:
   - npm install
   - npm install jshint
+  - export DISPLAY=:99.0 
+  - sh -e /etc/init.d/xvfb start
```

Comment:
Why?
Can we move these assets, deployment scripts, CI configs, etc. into another repos which pulls in ethereum as a submodule?  We're pulling in 5-15k+ LOC of JS every time small changes are made to those scripts and they're re-"compiled". Instead, a separate project would need to pull in far fewer files from cpp-ethereum, and changes are made, it doesn't require a 5k+ LOC PR.